src/core/src/Gsim.cc

```diff
@@ -544,6 +543,7 @@ void Gsim::MakeEvent(const G4Event *g4ev, DS::Root *ds) {
   int numPE = 0;
 
   for (int ipmt = 0; ipmt < hitpmts->GetEntries(); ipmt++) {
+    double chargeScale = 1;
```

Comment:
Can we move this down to l. 558? I believe GetChannelStatus should set the default value via the ratdb entry you added.

Reply:
done :)